CLAUDE.md

```diff
@@ -0,0 +1,37 @@
+# Claude Code Notes for LinkML
+
+## Project Structure
+
+This is a UV workspace monorepo that publishes **two PyPI packages**:
+
+| Package | Source | Tests | PyPI |
+|---------|--------|-------|------|
```

Comment:
Just chiming in.  This line of dashes is a lot of tokens,  and it doesn't actually add any info for the agent to act on.   I suggest dropping it.